Source/Scene/Cesium3DTileFeature.js

```diff
@@ -46,7 +46,7 @@ define([
         this._batchTable = content.batchTable;
         this._batchId = batchId;
         this._color = undefined;  // for calling getColor
-
+        this._tile = content._tile;
```

Comment:
`_tile` and `_content` will need to lose the underscores to be consistent with the pick object sent to `Model` and the pick object in `PointCloud3DTileContent`. Add doc above them like for `this.primitive`.